manifests/opsmanager.pp

```diff
@@ -0,0 +1,62 @@
+# This installs Ops Manager. See README.md for more info.
+
+class mongodb::opsmanager (
+  String $user                      = $mongodb::params::user,
```

Comment:
For strings, can you use `String[1]`? That enforces a minimal string length of 1.